queries/pioasm/injections.scm

```diff
@@ -0,0 +1,11 @@
+ [ (line_comment) (block_comment) ] @comment
+
+((code_block
+  (code_block_language) @_language
+  (code_block_body) @c)
+ (#eq? @_language "c-sdk"))
+
+((code_block
+  (code_block_language) @language
+  (code_block_body) @content)
+ (#eq? @language "python"))
```

Comment:
The `#eq?` predicate isn't needed anymore now that you are using `@language/@content`